tests/test_acasclient.py

```diff
@@ -2717,8 +2718,45 @@ def test_055_get_sdf_by_bulk_load_file(self):
         self.assertIn('CMPD-0000001-001', sdf_string)
         self.assertIn('$$$$', sdf_string)
 
+    @requires_basic_cmpd_reg_load
+    def test_056_additional_assay_scientists(self):
+        """Testing additional assay scientists."""
+
+        # Test getting the current assay scientists
+        assay_scientists = self.client.get_assay_scientists()
+
+        # Verify bob is in the list
+        self.assertEqual(1, len([x for x in assay_scientists if x['code'] == 'bob']))
+
+        # Create a new assay scientist code and name using uuid
+        new_assay_scientist_code = str(uuid.uuid4())
+        new_assay_scientist_name = str(uuid.uuid4())
+
+        # Test create new scientist
+        resp = self.client.create_assay_scientist(new_assay_scientist_code, new_assay_scientist_name)
+        self.assertEqual(resp['code'], new_assay_scientist_code)
+        self.assertEqual(resp['name'], new_assay_scientist_name)
+
+        # Verify that the new assay scientist is in the list
+        assay_scientists = self.client.get_assay_scientists()
+        self.assertEqual(1, len([x for x in assay_scientists if x['code'] == new_assay_scientist_code]))
+
+        # Verify we can load an experiment file with the new assay scientist
+        data_file_to_upload = get_basic_experiment_load_file(self.tempdir, scientist=new_assay_scientist_code)
 
-def get_basic_experiment_load_file_with_project(project_code, tempdir, corp_name = None, file_name = None):
+        # This uploads the experiment and verifies the upload was successful
+        result = self.experiment_load_test(data_file_to_upload, False)
+        
+        # Fetch the experiment
+        experiment_code = result['results']['experimentCode']
+        experiment_dict = self.client.get_experiment_by_code(experiment_code)
+
+        # Verify the scientist is the same as we specified
+        experiment = Experiment(experiment_dict)
+        self.assertEqual(experiment.scientist, new_assay_scientist_code)
+
+
+def get_basic_experiment_load_file(tempdir, project_code=None, corp_name=None, file_name=None, scientist=None):
```

Comment:
Refactored this method and it's uses so that I could incorporate scientist replacement and have the name reflect more accurately what it does (not dedicated to just updating the project).